### PR TITLE
Fix ansible-lint warnings on latest version

### PR DESCRIPTION
**Pull request by Pythyu**
Fix ansible-lint warnings on latest version for the collection

### CODE REVIEW

File: tasks/pkg-debian.yml [hunk 1]
```diff
@@ -42,7 +42,7 @@
     - key: 5F1E256061D813B125E156E8E6266D4AC0962C7D
       value: "{{ datadog_apt_key_url_new }}"
     - key: D18886567EABAD8B2D2526900D826EB906462314
-      value: "{{ datadog_apt_key_url_new }}"
+      value: "{{ datadog_apt_key_url_new }}"
```
Comment (spencergilbert) [inline tasks/pkg-debian.yml:45]: 👀 what's the diff here?

**Review by spencergilbert — APPROVED** [review]

File: tasks/pkg-debian.yml (hunk 1 above)
Comment (Pythyu) [reply tasks/pkg-debian.yml:45]: Don't ask me, ask `ansible-lint` I just copy pasted what fix he suggested to me 😆 
idk

Comment (chouquette) [reply tasks/pkg-debian.yml:45]: Non breaking space VS regular space

Comment (f4usto) [reply tasks/pkg-debian.yml:45]: \t \s ?

**Review by f4usto — APPROVED** [review]

**Pull request merged**